### PR TITLE
Decouple Domains and Redirects

### LINKED ISSUE #3600: Increase Visibility for Additional Redirect Scenarios
Re: [Domains and Redirects](https://pantheon.io/docs/domains/)

Priority (Low‚ Medium‚ High): Medium

## Issue Description:
Users and CSE find it difficult to locate documentation for additional redirect scenarios, given that it's currently hidden within a dropdown panel

## Suggested Resolution
- Extract the content from the dropdown panel either 1) on this page or 2) in it's own separate doc 

Keep in mind that the only redirect we recommend implementing as part of the launch process is to redirect to HTTPS and a primary domain. All other scenarios are considered "optional" or "as needed" when it makes sense for the site. So they should be covered in a way that does not distract users while going live

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #3600 

## Effect
PR includes the following changes:
- Decouple redirect content within **Domains and Redirects** onto it's own page at `/docs/redirects/`
- Consolidate **Using PHP as an htaccess Alternative** content within redirect doc at `/docs/redirects/`
- Update page titles and links throughout as needed 
- Sanitize redirect snippets for consistent logic (e.g., new relic transactions) and spacing 
- Improve page titles and descriptions 

## Post Launch
- [ ] Exclude `/docs/htaccess` from search index
- [ ] Add `/docs/redirects` to search index
- [ ] Reindex `/docs/domains` within search index
- [ ] Make sure existing redirect login on pantheon.io does not conflict with new page at `/docs/redirects` (we used to have a doc page here so there's likely a redirect in place that needs to be removed)
- [ ] Notify cse of change on deployment since this is frequently used content
- [ ] Update Status Report
- [ ] Archive from **Done** in Waffle

### CODE REVIEW

**Comment by eabquina** [comment]
@alexfornuto +1 for this.  This may be a home for app-specific redirect issues too.

**Comment by lauren-kelly** [comment]
:+1:

**Comment by alexfornuto** [comment]
Thanks to everyone who reviewed and commented. I'm going to call "Technical Review" complete since none of the actual code has changed, and all the feedback from @pantheon-systems/cse has been positive. Ready for Copy Review

File: source/_docs/redirects.md [hunk 1]
```diff
@@ -1,94 +1,122 @@
...
+## Considerations
+### PHP vs htaccess   
+Pantheon does not support managing redirects in `.htaccess` files, since they are ignored by [NGINX](https://www.nginx.com/resources/wiki/#){.external} for reduced resource consumption and increased efficiency. This configuration is standard across all Pantheon sites, and modifications to the `nginx.conf` file are not supported.
 
+Using `.htaccess` is generally not recommended - even for sites running  [Apache](https://httpd.apache.org/docs/trunk/howto/htaccess.html#when){.external}. Instead, we suggest handling redirects in PHP within your site's configuration file. Some advantages of redirecting via PHP instead of `.htaccess` include:
+
+- Logic and decisions can be made that a web server would have no context for, as it's executable code with application state awareness. Conditional logic, regular expressions, and much more are possible.
+- Configuration tends to be more maintainable as Drupal and WordPress developers are typically literate in PHP, but very few people are naturally fluent in Apache2 rewrite rules and conditions.
```
[the remark this answers is not in the document]
Comment (rachelwhitton) [reply source/_docs/redirects.md]: This was just copy/paste to preserve content from the deleted page. Copy edits welcomed!

File: source/_docs/redirects.md [hunk 2]
```diff
@@ -1,94 +1,122 @@
...
+- Since `settings.php` and `wp-config.php` are parsed very early in the bootstrap process, redirects like this are "cheap" with low overhead. If you use a 301 redirect, Varnish will cache it as well.
```
[the remark this answers is not in the document]
Comment (rachelwhitton) [reply source/_docs/redirects.md]: +1

**Pull request merged**